File: app/src/main/java/com/vkpapps/thunder/model/SerializedMessage.java
package com.vkpapps.thunder.model;

/**
 * @author VIJAY-PATIDAR
 */
public class SerializedMessage {
    private int messageType;
    private String messageBody;

    public SerializedMessage(int messageType, String messageBody) {
        this.messageType = messageType;
        this.messageBody = messageBody;
    }

    public int getMessageType() {
        return messageType;
    }

    public void setMessageType(int messageType) {
        this.messageType = messageType;
    }

    public String getMessageBody() {
        return messageBody;
    }

    public void setMessageBody(String messageBody) {
        this.messageBody = messageBody;
    }
}

File: app/src/main/java/com/vkpapps/thunder/model/SerializedMessage.kt
package com.vkpapps.thunder.model

import com.google.gson.annotations.SerializedName

/**
 * @author VIJAY-PATIDAR
 */
class SerializedMessage {
    @SerializedName("user")
    var user: User? = null

    @SerializedName("requestInfo")
    var requestInfo: RequestInfo? = null

    @SerializedName("fileRequest")
    var fileRequest: FileRequest? = null

    constructor(fileRequest: FileRequest?) {
        this.fileRequest = fileRequest
    }

    constructor(user: User?) {
        this.user = user
    }

    constructor(requestInfo: RequestInfo?) {
        this.requestInfo = requestInfo
    }

}